notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>You should put:</p><ul><li>at the beginning of this notebook: "Lead Author: Adele Myers", and you can add a link to your researcher's webpage.</li><li>and just below "Inspired by: Guigui, Miolane, Pennec, 2022. Introduction to Riemannian Geometry and Geometric Statistics: from basic theory to implementation with Geomstats."</li></ul><p> </p><p>I would remove the "Geomstats is one of the first software packages of its kind" (so that we are 100% sure that this cannot be interpreted as "bragging").</p><p> </p><p>That uses concepts from Riemannian Geometry</p><p>-&gt; from Differential Geometry and Riemannian Geometry</p><p>(before we introduce a Riemannian metric, we are actually doing differential geometry, and not Riemannian geometry).</p><p> </p><p>only people with prior knowledge of Riemannian geometry could analyze data on manifolds</p><p>-&gt; Analyzing data on manifolds could be a difficult task for people without prior knowledge of differential geometry</p><p> </p><p>NIT: prioro -&gt; prior</p><p>NIT: Word towards? Do you mean work towards?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>